.github/actions/helm-publish-chart/action.yml

```diff
@@ -0,0 +1,100 @@
+name: "Add package to helm repository"
+description: "Add a new package to the helm repository, updates the index file and commit the change"
+inputs:
+  chart-package:
+    description: "The name of the packaged chart been added to the repository. I.e. common-x.y.z.tgz"
+    required: true
+  helm-charts-repo:
+    description: "The name of the repository where the package will be added"
+    required: true
+  helm-charts-repo-branch:
+    description: "The name of the branch where the package will be added"
+    required: true
+  helm-charts-repo-subfolder:
+    description: "The name of the subfolder inside the charts repository where the package should be added"
+    required: false
+  helm-charts-repo-base-url:
+    description: "Helm chart repo base url"
+    required: false
+  token:
+    description: "The github token"
+    required: true
+  git-username:
+    description: "The username to use for git commit"
+    required: false
+    default: ""
+  do-push:
+    description: "If 'true', changes will be pushed ('false' can be used for PRs)"
+    default: 'true'
+    required: false
+
+runs:
+  using: composite
+  steps:
+
+    - name: Generate random path for helm repository checkout
+      shell: bash
+      run: echo "CHECKOUT_PATH=$(uuidgen)" >> $GITHUB_ENV
+
+    - name: Generate random path for helm package
+      shell: bash
+      run: echo "RANDOM_PATH=$(uuidgen)" >> $GITHUB_ENV
+
+    - name: Checkout charts repository
+      uses: actions/checkout@v3
+      with:
+        path: ${{ env.CHECKOUT_PATH }}
+        repository: ${{ inputs.helm-charts-repo }}
+        ref: ${{ inputs.helm-charts-repo-branch }}
+        token: ${{ inputs.token}}
+
+    - name: Build base URL option
+      id: build-base-url-option
+      env:
+        CHARTS_REPO_BASE_URL: ${{ inputs.helm-charts-repo-base-url }}
+      shell: bash
+      run: |
+        if [ -n "$CHARTS_REPO_BASE_URL" ]
+        then
+            echo "result=--url $CHARTS_REPO_BASE_URL" >> $GITHUB_OUTPUT
+        fi
+
+    - name: Build destination path
+      id: build-destination-path
+      shell: bash
+      env:
+        CHARTS_REPO_SUBFOLDER: ${{ inputs.helm-charts-repo-subfolder }}
+      run: |
+        if [ -n "$CHARTS_REPO_SUBFOLDER" ]
+        then
+            echo "result=$CHARTS_REPO_SUBFOLDER" >> $GITHUB_OUTPUT
+        else
+            echo "result=." >> $GITHUB_OUTPUT
+        fi
+
+    - name: Add package to Helm repository
```

Comment:
I have a remaining issue here since I get the following diff here:
```
-    - https://kubernetes-charts.alfresco.com/incubator/alfresco-process-application-7.7.0.tgz
+    - https://kubernetes-charts.alfresco.com/alfresco-process-application-7.7.0.tgz
```
see https://github.com/Alfresco/alfresco-process-application-deployment/actions/runs/3418179132/jobs/5690172010

Reply:
Fixed by latest changes -> I will rework the commits to make the changes to this more explicit in one single commit